keanu-project/src/main/java/io/improbable/keanu/tensor/bool/BooleanTensor.java

```diff
@@ -26,6 +26,8 @@ static BooleanTensor placeHolder(int[] shape) {
         return new SimpleBooleanTensor(shape);
     }
 
+    BooleanTensor reshape(int... newShape);
```

Comment:
Can we make reshape part of the Tensor interface?

` Tensor<T> reshape(int... newShape);`

Reply:
Yep, added